cocos/native-binding/index.ts

```diff
@@ -1401,4 +1401,51 @@ export declare namespace native {
          */
         export const argv: Readonly<string[]>;
     }
+
+    /**
+     * @en A module used to detect blocking execution of the gameTick procedure.
+     * @zh 用于检测 gameTick 执行阻塞的模块
+     */
+    namespace blockingDetection {
```

Comment:
`blockingDetection ` is too long for a namespace.

Reply:
What about `blkDct` ?